charts/testkube-api/templates/deployment.yaml

```diff
@@ -310,7 +310,7 @@ spec:
               value: "{{ .Values.cloud.uiUrl }}"
             {{- end}}
             {{- if not .Values.cloud.tls.enabled }}
-            - name: TESTKUBE_PRO_TLS_INSECURE
+            - name: TESTKUBE_CLOUD_TLS_INSECURE
```

Comment:
Should it be renamed, considering that we have renamed all the variables to `TESTKUBE_PRO`?

<img width="1365" alt="Zrzut ekranu 2024-02-23 o 16 52 04" src="https://github.com/kubeshop/helm-charts/assets/3843526/3d4509be-dca7-4bf0-aa8b-3ec5e1f0259d">
<img width="921" alt="Zrzut ekranu 2024-02-23 o 16 52 22" src="https://github.com/kubeshop/helm-charts/assets/3843526/825b493a-3434-4017-94f9-cb6569d1b1db">
<img width="1231" alt="Zrzut ekranu 2024-02-23 o 16 52 50" src="https://github.com/kubeshop/helm-charts/assets/3843526/5a0dd13d-6e2c-4ecb-8670-3ba2a6349a54">

We are specifically using this one now.

<img width="1589" alt="Zrzut ekranu 2024-02-23 o 16 53 02" src="https://github.com/kubeshop/helm-charts/assets/3843526/34869115-5a59-4a41-a455-137bc3bb93dc">

Reply:
hm, I didn't see `TESTKUBE_PRO_TLS_INSECURE ` anywhere in testkube-cloud-api repo. At least, it didn't work when I used it in a chart.

Reply:
It's not in `testkube-cloud-api` repo - it's in `testkube` repository.

Reply:
nvm, I used an old agent, closing this